src/tests.zig

```diff
@@ -2411,3 +2411,250 @@ test "namecall" {
     lua.pop(-1);
     try expectEqual(6, s);
 }
+
+test "toAny int" {
```

Comment:
I'm not 100% against the way you set this up, but my personal preference is to combine related tests. So I would probably write all of the `toAny` tests in a single test block. 

But if you feel like this is better I'm okay with that

Reply:
The reason I did that is because the function can work fine for some datatypes but be completely broken on others. So I split it into different tests because it seemed like whether popping an int failed or not was pretty unrelated to whether popping an odd slice type would fail. 

But I could also combine them is that is the standard here

Reply:
Personally I prefer them combined. But you can leave them separate. Maybe I'll combine them in the future, or leave them separate.. 🤷‍♂️

Reply:
I can combine similar tests then

Reply:
Thanks